### DISCUSSION TITLE
Controller Upgrade Recommendation?

### DISCUSSION THREAD

**Discussion by Archer36** [zwave-js/zwave-js#6638] [Q&A] [answered]
Hello, I'm currently using a Zooz Z-Wave Plus S2 Stick ZST10 that I purchased back in 2019, I believe its a Z-Wave 500 device. My Z-Wave mesh consists of 56 nodes of different generations and performance seems decent. It's running on a Raspberry Pi 3 Model B+ with Z-Wave JS UI running from a container, which then connects into my Home Assistant hosted on another server. To my knowledge the only thing that doesn't work well with my current controller is soft resets.

There are obviously 700 and 800 controllers now and I'm wondering if I should be looking to upgrade to a newer controller and if so which one should I look at (USB based). From what I've seen on the discussion boards here seems like 800 series are still experimental. 

Thank you!

**Comment by nmpu** [comment]
It sure sounds like you already have a 700 series?

I recently switched from 500 to 800 series. Based on [my experience](https://community.home-assistant.io/t/is-there-a-way-for-z-wave-js-to-access-a-remote-usb-stick-over-ip/668973/9), you're not going to see any noticeable difference. I suspect a [better antenna](https://community.hubitat.com/t/zooz-zst10-usb-stick-external-antenna/118808) would offer the greatest improvement.

**Comment by AlCalzone** [comment, marked as answer]
If the question is still relevant, I only recommend 800 series right now, since it has superior range in all my tests. The Zooz ZST39 (800 series) is pretty good. If you want to get your hands dirty with programming it and possibly printing an enclosure, the [800 series devkit](https://www.silabs.com/development-tools/wireless/z-wave/z-wave-800-dev-kit?tab=overview) is even better. 700 series has been terrible in my experience.
